db/beer.go

```diff
@@ -0,0 +1,61 @@
+package db
+
+//Beer is an entity that represents a record in the database
+type Beer struct {
```

Comment:
Beer should have it's own package.
That will allow us to get everything related to the Beer on the same place and make it portable if needed.